### ISSUE TITLE
Ally issues w/ Data Tracker Joyride / Tutorial Modal (for first time users)

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1042]
- [x] Reading the Explore Data page 	SR	Improper behavior is observed when user first time get into the Explore data page.	"ENVIRONMENT:
OS | Browser | Screen Reader
Window | Chrome Version 91.0.4472.77 (Official Build) (64-bit) | JAWS/NVDA
Window | Firefox Version: 87.0 (64-bit) | JAWS
PREREQUISITE:
Enable the respective screen reader.
STEPS TO REPRODUCE:
  1) Open this Url: https://healthequitytracker.org/
  2) Navigate to the Explore the data tab and select.
  3) Observe screen reader announcement and focus.
ACTUAL RESULT: Focus set onto the next button on the dialog popup, but screen reader is announcing background page content when the page is open for the first time.
EXPECTED RESULT: Focus should set on to the tutorial popup and screen reader should read the popup tutorial info to the user."	https://drive.google.com/file/d/1i8orHb2_aSR1Qm8aSyFI83yBIKD3_8pL/view?usp=sharing

- [x] Reading the Explore Data page 	SR	Redundant announcement is observed for the first tutorial popup dialog on Explore the data page.	"ENVIRONMENT:
OS | Browser | Screen Reader
ChromeBook | Chrome Version: 90.0.4430.218 (Official Build) (64-bit) | ChromeVox
Window | Chrome Version 91.0.4472.77 (Official Build) (64-bit) | JAWS/NVDA
Window | Firefox Version: 87.0 (64-bit) | JAWS
PREREQUISITE:
Enable the respective screen reader.
STEPS TO REPRODUCE:
  1) Open this Url: https://healthequitytracker.org/
  2) Navigate to the Explore the data tap and select.
  3) Observe screen reader announcement.
ACTUAL RESULT: Screen reader is reading all content of the dialog window twice which is not required.
EXPECTED RESULT: Screen reader should not read the dialog content twice."	

- [x] Reading the Explore Data page 	Color contrast	Insufficient color contrast for 'back' button.	"ENVIRONMENT:
OS | Browser | Screen Reader
macOS Big Sur:11.4|Version 90.0.4430.212 (Official Build) (x86_64) | VoiceOver
Tool Used: Color contrast Analyser
STEPS TO REPRODUCE:
  1) Open this Url: https://healthequitytracker.org/
  2) Navigate to the Explore the data tap and select.
  3) Select Next button on tutorial popup dialog.
  4) Verify the color contrast for 'back' button.
ACTUAL RESULT: Insufficient color contrast for back button
EXPECTED RESULT: Color contrast ratio for regular size text should be at least 4.5:1."

- [x] Reading the Explore Data page 	SR	User is not getting informed that there is a skip button on the tutorial popup.	"ENVIRONMENT:
OS | Browser | Screen Reader
macOS Big Sur:11.4|Version 90.0.4430.212 (Official Build) (x86_64) | VoiceOver
Window | Chrome Version 91.0.4472.77 (Official Build) (64-bit) | JAWS/NVDA
Window | Firefox Version: 87.0 (64-bit) | NVDA
PREREQUISITE:
Enable the respective screen reader.
STEPS TO REPRODUCE:
  1) Open this Url: https://healthequitytracker.org/
  2) Navigate to the Explore the data tap and select (for the first time)
  3) Observe screen reader announcement on the popup dialog.
ACTUAL RESULT: There are no indication to the user about the 'skip' button existed on the popup dialog.
EXPECTED RESULT: User should getting informed about the skip button."

- [x] Reading the Explore Data page 	SR(ChromeVox)	Improper announcement of content on tutorial popup dialog.	"ENVIRONMENT:
OS | Browser | Screen Reader
ChromeBook | Chrome Version: 90.0.4430.218 (Official Build) (64-bit) | ChromeVox
PREREQUISITE:
Enable the respective screen reader.
STEPS TO REPRODUCE:
  1) Open this Url: https://healthequitytracker.org/
  2) Navigate to the Explore the data tap and select (for the first time)
  3) Observe screen reader announcement on the popup dialog.
ACTUAL RESULT: Screen reader is announcing skip and next button as a normal text
EXPECTED RESULT: There should be some indication to the user to distinguish between normal text and buttons."

- [x] Reading the Explore Data page 	SR	User is not getting informed of number of tutorials popup.	"ENVIRONMENT:
OS | Browser | Screen Reader
ChromeBook | Chrome Version: 90.0.4430.218 (Official Build) (64-bit) | ChromeVox
Window | Chrome Version 91.0.4472.77 (Official Build) (64-bit) | JAWS/NVDA
Window | Firefox Version: 87.0 (64-bit) | JAWS/NVDA
PREREQUISITE:
Enable the respective screen reader.
STEPS TO REPRODUCE:
  1) Open this Url: https://healthequitytracker.org/
  2) Navigate to the Explore the data tap and select (for the first time).
  3) Observe screen reader announcement for next 1/4 button.
ACTUAL RESULT: The user is not getting informed about the number of tutorials popup. Screen reader is only announcing the 'Next' button and skipping 1/4. 
EXPECTED RESULT: User should getting informed about the number of tutorials popup (e.g., 1 out of 4. 2 out of 4)"

**Comment by benhammondmusic** [comment]
Should fix focus indicator color; looks like it's black on green in firefox, and double ring of blue and black on Edge. Should be white or other obvious color scheme color

![bar 100k 2](https://user-images.githubusercontent.com/41567007/137200508-2d06dc67-3cde-41b1-b537-f0f722ac7af4.png)
